docs/migrate/migrate-setup.rst

```diff
@@ -0,0 +1,182 @@
+Migrate setup from ``qiskit-ibmq-provider``
+==============================================
+
+This guide describes how to migrate code from the legacy IBMQ provider (`qiskit-ibmq-provider`) package to use Qiskit Runtime (`qiskit-ibm-runtime`). This guide includes instructions to migrate legacy runtime programs to the new syntax. However, the ability to use custom uploaded programs is pending deprecation, so these should be migrated to use primitives instead.  
+
+- For instructions to use Qiskit Runtime primitives, see the `migration guide <migrate-guide.html>`__.  
+- For further details about migrating from `qiskit-ibmq-provider` to `qiskit-ibm-runtime`, see `Migration guide from qiskit-ibmq-provider <https://qiskit.org/documentation/partners/qiskit_ibm_runtime/migrate_from_ibmq.html>`__.
```

Comment:
We should really combine the two. It's odd to have a "migrate from qiskit-ibmq-provider" guide pointing to another guide with the same title lol